ios/FluentUI.Demo/FluentUI.Demo/Demos/CardNudgeDemoController.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class CardNudgeDemoController: UITableViewController {
+    enum CardControls: Int, CaseIterable {
+        case mainIcon
+        case subtitle
+        case accentIcon
+        case accentText
+        case dismissButton
+        case actionButton
+
+        var text: String {
+            switch self {
+            case .mainIcon:
+                return "Main Icon"
+            case .subtitle:
+                return "Subtitle"
+            case .accentIcon:
+                return "Accent Icon"
+            case .accentText:
+                return "Accent"
+            case .dismissButton:
+                return "Dismiss Button"
+            case .actionButton:
+                return "Action Button"
+            }
+        }
+    }
+
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        super.init(style: .plain)
+
+        self.prepareCardNudgeViews()
+    }
+
+    required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: CardNudgeDemoController.controlReuseIdentifier)
```

Comment:
Yep! Otherwise you get this error on load:
```
Thread 1: "unable to dequeue a cell with identifier controlReuseIdentifier - must register a nib or a class for the identifier or connect a prototype cell in a storyboard"```